TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config

```diff
@@ -139,8 +139,8 @@ train_config: {
       learning_rate: {
         exponential_decay_learning_rate {
           initial_learning_rate: 0.01
-          decay_steps: 400
-          decay_factor: 0.95
+          decay_steps: 300
```

Comment:
and this could be 500

Reply:
I found 300 is better according to the loss behaviours.

Reply:
even better than 400